test/src/webview.js

```diff
@@ -1,11 +1,111 @@
 import test from 'socket:test'
 
+import { fetch, Headers, Request, Response } from 'socket:fetch'
+import { URL, URLPattern, URLSearchParams } from 'socket:url'
+import { ApplicationURLEvent } from 'socket:internal/events'
+import Notification from 'socket:notification'
+import { Buffer } from 'socket:buffer'
+
+import {
+  File,
+  FileSystemHandle,
+  FileSystemFileHandle,
+  FileSystemDirectoryHandle,
+  FileSystemWritableFileStream
+} from 'socket:fs/web'
+
+import {
+  showDirectoryPicker,
+  showOpenFilePicker,
+  showSaveFilePicker
+} from 'socket:internal/pickers'
+
 const loaded = new Promise((resolve) => {
-  window.addEventListener('load', () => {
+  globalThis.addEventListener('load', () => {
     resolve(true)
   }, { once: true })
 })
 
-test('window.onload', async (t) => {
-  t.ok(await loaded, 'window.onload called')
+test('globalThis.onload', async (t) => {
+  t.ok(await loaded, 'globalThis.onload called')
+})
+
+test('globalThis.isSocketRuntime', (t) => {
+  t.ok(globalThis.isSocketRuntime === true, 'globalThis.isSocketRuntime')
+})
+
+test('globalThis.Buffer', (t) => {
+  t.ok(typeof globalThis.Buffer === 'function' && globalThis.Buffer === Buffer, 'globalThis.Buffer')
```

Comment:
I'd change the message here and in other places to something like "Buffer is available on globalThis" and "globalThis.URL is normalized"

Reply:
we can improve this later 🙏🏼